src/main/java/org/ideoholic/curium/model/account/service/AccountService.java

```diff
@@ -1072,10 +1072,10 @@ public void getAllLedgers() {
 	}
 
 
-	public boolean getIncomeStatement() {
+	public boolean getIncomeStatement(IncomeStatementDto incomeStatementDto) {
 		
-		String fromDate = DataUtil.dateFromatConversionDash(DataUtil.emptyString(request.getParameter("fromdate")));
-		String toDate = DataUtil.dateFromatConversionDash(DataUtil.emptyString(request.getParameter("todate")));
+		String fromDate = DataUtil.dateFromatConversionDash(DataUtil.emptyString(incomeStatementDto.getFromDate()));
+		String toDate = DataUtil.dateFromatConversionDash(DataUtil.emptyString(incomeStatementDto.getToDate()));
 		
 		if(httpSession.getAttribute(BRANCHID)!=null) {
```

Comment:
Remove `httpSession.getAttribute(BRANCHID)` and add `incomeStatementDto.getBranchId()`

Reply:
Made the changes sir, I have changed the data type in dto class from 'int' to 'Integer' as int cannot accept null.